unblob/cli.py

```diff
@@ -185,6 +185,20 @@ def cli(
 cli.context_class = UnblobContext
 
 
+def get_exit_code_from_reports(reports: List[Report]) -> int:
+    exit_code_map = {
+        Severity.ERROR: 1,
+        Severity.WARNING: 0,
+    }
+    severities = {report.severity for report in reports}
+
+    for severity, exit_code in exit_code_map.items():
```

Comment:
`exit_code_map` is a map, but its only usage is iterating over it.
I have also grown up with unordered dicts, where looping over items of dicts is ambiguous.
I know under py3 dicts are ordered, but still it reads wrong.
```py
exit_code_map = [
        (Severity.ERROR, 1),
        (Severity.WARNING, 0),
    ]
```
is better.

Reply:
An alternative implementation could be: take the set of severities, take the largest, map that value to exit code.

Reply:
Dicts are designed to keep insertion order from python3.7. Since this project is python3.8+, I think this is fine.

However I did change the suggested for `exit_code_map` ( I renamed it to `severity_to_exit_code` since it become a `list` ...), if you think this is easier to understand / more clear what is happening, this is fine by me.